lesson1/task03/Calculator.java

```diff
@@ -0,0 +1,77 @@
+package com.company;
+
+/**
+ * Arithmetic with numbers
+ * @author Vaitsakhouski Raman
+ * @version 1.0
+ * @since 01-10-2016
+ */
+public class Calculator {
+
+    /**
+     * Calculates 2 numbers and prints result to console
+     *
+     * @param number_1 - first number
+     * @param number_2 - second number
+     * @param a link to number_1
+     * @param b link to numbe_2
+     */
+    public double a;
+    public double b;
+    public Calculator(double number_1, double number_2) {
+        a=number_1;
+        b=number_2;
+        System.out.println(number_1 + "+" + number_2 + "=" + sum(number_1,number_2));
+        System.out.println(number_1 + "-" + number_2 + "=" + subtr(number_1,number_2));
+        System.out.println(number_1 + "*" + number_2 + "=" + multipl(number_1,number_2));
+        if(number_2==0) {

```

Comment:
Invalid double comparison http://www.java67.com/2015/09/float-and-double-value-comparison-in-java-use-relational.html